pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/channel/ServiceUnitStateChannelImpl.java

```diff
@@ -815,6 +819,15 @@ private void handleOwnEvent(String serviceUnit, ServiceUnitStateData data) {
 
     private void handleAssignEvent(String serviceUnit, ServiceUnitStateData data) {
         if (isTargetBroker(data.dstBroker())) {
+            if (disablePubOwnedEvent) {
+                log.info("Skip assigning self({}) as the owner after cleanOwnerships", serviceUnit);
+                final var getOwnerRequest = getOwnerRequests.remove(serviceUnit);
```

Comment:
This will make other brokers to wait for the Owned event. I think whatever ongoing state change should follow the protocol without interruption.

I think better place to check this "disablePubOwnedEvent" state in before publishing the state change command.

- publishAssignEventAsync
- publishOverrideEventAsync
- publishUnloadEventAsync
- publishSplitEventAsync

We already have `validateChannelState` func to check the channel state, and I think we can additionally consider "disablePubOwnedEvent" state there.

Reply:
I used the `Disabled` state instead and disabled all events except the override events (whose force field is true)